tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1040,112 +1059,118 @@ def test_continuous_reads_during_pod_restart(self):
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    skipwrite=True, skipcleanup=True,
+                                                    nclients=10, nsamples=10)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 3: Performed WRITEs/READs-Verify with variable sizes objects.")
 
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=remain_pod_list, fail=False)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 4: Services of pod are in online state")
+        LOGGER.info("Step 4: Perform WRITEs/READs-Verify with variable object sizes.")
+        if CMN_CFG["dtm0_disabled"]:
+            self.test_prefix = 'test-34073-1'
 
-        LOGGER.info("Step 5: Perform WRITEs with variable object sizes. (0B - 512MB(VM)/5GB(HW))")
-        users = self.mgnt_ops.create_account_users(nusers=1)
-        self.test_prefix = 'test-34073'
-        self.s3_clean.update(users)
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                     log_prefix=self.test_prefix, skipread=True,
                                                     skipcleanup=True, nclients=10, nsamples=10)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 5: Performed WRITEs with variable sizes objects.")
+        LOGGER.info("Step 4: Performed WRITEs/READs-Verify with variable object sizes.")
 
-        LOGGER.info("Step 6: Perform READs and verify DI on the written data in background")
+        LOGGER.info("Step 5: Perform READs and verify DI on the data in background")
+        self.test_prefix = 'test-34073'
         args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
                 'nclients': 1, 'nsamples': 10, 'skipwrite': True, 'skipcleanup': True,
                 'output': output}
-
         thread = threading.Thread(target=self.ha_obj.event_s3_operation,
                                   args=(event,), kwargs=args)
         thread.daemon = True  # Daemonize thread
         thread.start()
-
-        LOGGER.info("Step 6: Successfully started READs and verified DI on the written data in "
+        if CMN_CFG["dtm0_disabled"]:
+            self.test_prefix = 'test-34073-1'
+            args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
+                    'nclients': 1, 'nsamples': 10, 'skipwrite': True, 'skipcleanup': True,
+                    'output': output1}
+            thread1 = threading.Thread(target=self.ha_obj.event_s3_operation,
+                                       args=(event,), kwargs=args)
+            thread1.daemon = True  # Daemonize thread
+            thread1.start()
```

Comment:
So we will be starting 2 threads ? if dtm0 is true

Reply:
We will be starting 2 threads if dtm is disabled.